### DISCUSSION TITLE
[FEATURE] Integrate pawapay mobile money payment services into HyperSwitch

### DISCUSSION THREAD

**Discussion by dave-evans-pawapay** [juspay/hyperswitch#3226] [Ideas / Feature Requests]
pawaPay is a leading mobile money service provider in Africa (www.pawapay.io).  We would be interested in building a connector between hyperswitch and pawapay to enable hyperswitch merchants to take payments in Africa

Am planning to allocate engineering resource on this in Q1 2024

**Comment by manojradhakrishnan** [comment]
Thanks @dave-evans-pawapay 

Please keep us posted. And you may add your engineering team to this thread for any support required during the process.
